README.adoc

```diff
@@ -127,6 +127,26 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
+
+- HTTP Proxy Settings +
+In an environment where HTTP traffic needs pass through a HTTP proxy, Docker
+and OpenShift need to be aware of the required proxy settings in order to function.
+`http_proxy`, `http_proxy_user` and `http_proxy_password` configurations need to be
+specified through Vagrantfile. You can do so via:
+
++
+----------------------------------------------------------------------------
+config.servicemanager.http_proxy = <Proxy URL>
+config.servicemanager.http_proxy_user = <Proxy user name>
+config.servicemanager.http_proxy_password = <Proxy user password>
+----------------------------------------------------------------------------
++
+
+When these settings are applied, they are passed through to the Docker and
+OpenShift service. `http_proxy_user` and `http_proxy_password` can be omitted in an
+unauthenticated proxy environment.
+
```

Comment:
@hferentschik Will create a new issue since we have not been considering that use case properly. Even current hook behavior works or not I don't know ATM. I think it will not since we attached to `vagrant up` action.